pages/scripts/stats.js

```diff
@@ -18,6 +18,29 @@ var timeoverride = null;
 var timeperiod;
 var unit = null;
 
+chrome.manifest = (function() {
```

Comment:
please can we put this in a shared library? just like shared_job_code.js etc above?